coding.go

```diff
@@ -43,12 +44,13 @@ func (d *safeBlockDecoder) Decode(block blocks.Block) (Node, error) {
 
 	if ok {
 		return decoder(block)
-	} else {
-		// TODO: get the *long* name for this format
-		return nil, fmt.Errorf("unrecognized object type: %d", ty)
 	}
+
+	// TODO: get the *long* name for this format
+	return nil, fmt.Errorf("unrecognized object type: %d", ty)
 }
 
+// DefaultBlockDecoder is a safeBlockDecoder with no decoders registered
```

Comment:
`safeBlockDecoder` isn't exported so I wouldn't mention it by here. I'd just say "DefaultBlockDecoder is the block decoder used by `Decode`" and maybe mention that (a) it's threadsafe and (b) users should register block decoders with it.